### PR TITLE
lib: Remove custom key navigation from TypeaheadSelect

### PR DESCRIPTION
**Pull request by mvollmer**
It's not helping.

### CODE REVIEW

File: pkg/playground/react-demo-multi-typeahead.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import { MultiTypeaheadSelect, MultiTypeaheadSelectOption } from "cockpit-components-multi-typeahead-select";
+
+const MultiTypeaheadDemo = ({ options } : { options: MultiTypeaheadSelectOption[] }) => {
+    const [notFoundIsString, setNotFoundIsString] = useState(false);
+    const [selected, setSelected] = useState<(string | number)[]>([]);
+    const [toggles, setToggles] = useState(0);
+    const [changes, setChanges] = useState(0);
+
+    function add(val: string | number) {
+        setSelected(selected.concat([val]));
```
Comment (cockpituous) [inline pkg/playground/react-demo-multi-typeahead.tsx:35]: These 2 added lines are not executed by any test.

File: pkg/playground/react-demo-multi-typeahead.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    function rem(val: string | number) {
+        setSelected(selected.filter(v => v != val));
```
Comment (cockpituous) [inline pkg/playground/react-demo-multi-typeahead.tsx:39]: These 2 added lines are not executed by any test.

File: pkg/playground/react-demo-multi-typeahead.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    return (
+        <div>
+            <MultiTypeaheadSelect
+                id='multi-typeahead-widget'
+                placeholder="Select flavors"
+                isScrollable
+                noOptionsFoundMessage={notFoundIsString ? "Not found" : val => cockpit.format("'$0' not found", val) }
```
Comment (cockpituous) [inline pkg/playground/react-demo-multi-typeahead.tsx:48]: This added line is not executed by any test.

File: pkg/playground/react-demo-multi-typeahead.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+                options={options}
+                selected={selected}
+                onAdd={add}
+                onRemove={rem}
+                onToggle={() => setToggles(val => val + 1)}
+                onInputChange={() => setChanges(val => val + 1)}
```
Comment (cockpituous) [inline pkg/playground/react-demo-multi-typeahead.tsx:54]: These 2 added lines are not executed by any test.

File: pkg/playground/react-demo-multi-typeahead.tsx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+            />
+            <div>Selected: <span id="multi-value">{JSON.stringify(selected)}</span></div>
+            <div>Toggles: <span id="multi-toggles">{toggles}</span></div>
+            <div>Changes: <span id="multi-changes">{changes}</span></div>
+            <Checkbox
+                id="notFoundIsStringMulti"
+                label="notFoundIsString"
+                isChecked={notFoundIsString}
+                onChange={(_event, checked) => setNotFoundIsString(checked)}
```
Comment (cockpituous) [inline pkg/playground/react-demo-multi-typeahead.tsx:63]: This added line is not executed by any test.

**Comment by mvollmer** [comment]
This only works with the PF coming in #21399. Before that, the text input keeps its focus and the template does indeed handle all key presses for the menu navigation.

**Pull request closed without merging**